doc/sphinx/source/input.rst

```diff
@@ -252,6 +252,8 @@ A list of the datasets for which a CMORizers is available is provided in the fol
 | CERES-SYN1deg                | rlds, rldscs, rlus, rluscs, rlut, rlutcs, rsds, rsdscs, rsus, rsuscs, rsut, rsutcs (3hr)             |   3  | NCL             |
 |                              | rlds, rldscs, rlus, rlut, rlutcs, rsds, rsdt, rsus, rsut, rsutcs (Amon)                              |      |                 |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
+| CFSR                         | clt, hus, hur, prw, rlut, rlutcs, rsut, rsutcs (3hr)                                                 |   2  | Python          |
```

Comment:
It would be good to add a note that the CMORized data can be downloaded from the CREATE-IP project on ESGF so users of ESMValTool do not need to CMORize the data themselves if they want to use it in a recipe.